pkg/templates/kubeadm/kubeadm.go

```diff
@@ -43,6 +48,15 @@ type Kubedm interface {
 }
 
 // New constructor
```

Comment:
I wonder how the old constructor looked like.

Reply:
Uh... good question 😅